certbot-dns-google/certbot_dns_google/_internal/dns_google.py

```diff
@@ -82,7 +82,7 @@ def __init__(self, account_json=None, dns_api=None):
         scopes = ['https://www.googleapis.com/auth/ndev.clouddns.readwrite']
         if account_json is not None:
             try:
-                credentials = ServiceAccountCredentials.from_json_keyfile_name(account_json, scopes)
+                credentials = Credentials.from_service_account_file(account_json, scopes)
```

Comment:
So, the error here from `tox -e lint` is:

    certbot-dns-google/certbot_dns_google/_internal/dns_google.py:85:30: E1121: Too many positional arguments for classmethod call (too-many-function-args)

Looking at https://google-auth.readthedocs.io/en/master/reference/google.oauth2.service_account.html#google.oauth2.service_account.Credentials, `scopes` is now a keyword argument, rather than a positional one. So this line should be:

```suggestion
                credentials = Credentials.from_service_account_file(account_json, scopes=scopes)
```